pkgs/tools/misc/ollama/default.nix

```diff
@@ -68,6 +68,10 @@ let
       cudaPackages.cudatoolkit
       cudaPackages.cuda_cudart
     ];
+    postBuild = ''
+      rm "$out/lib64"
```

Comment:
If it's a directory, shouldn't we use `-f` or eventually `-rf` ?

Reply:
it's a symlink